lib/site_prism/waiter.rb

```diff
@@ -18,5 +18,11 @@ def self.default_wait_time
       warn 'default_wait_time is now deprecated. This will be removed in an upcoming release.'
       Capybara.default_max_wait_time
     end
+
+    def self.wait_with_capybara(timeout: nil, or_raise: nil)
+      timeout = Capybara.default_max_wait_time if timeout.nil?
+      raise or_raise unless (result = Capybara.using_wait_time(timeout) { yield }) || !or_raise
```

Comment:
can we turn this into a nicer multi-line conditional to improve legibility

```
result = Capybara.using....

if result
  result
else
  raise or_raise
end
```

Reply:
```ruby
result = Capybara.using....
return result if result
raise or_raise if or_raise
```
How about this?

Reply:
Yeah, you've got a good point there, that one's a bit of a head scratcher.